### PR TITLE
Lessons learned managing GitHub Actions and Firecracker

### PR DESCRIPTION
**Pull request by alexellis**
Title: Lessons learned managing GitHub Actions and Firecracker

### CODE REVIEW

File: posts/2023-03-31-managing-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+GitHub is too big to fail. Let's hope they smooth out these bumps.
+
+## Lesson 3 - Webhooks can be unreliable
+
+There's no good API to collect this historical data at the moment but we do have an open-source tool ([self-actuated/actions-usage](https://github.com/self-actuated/actions-usage)) we give to customers to get a summary of their builds before they start out with us.
+
+So we mirror a summary of job events from GitHub into our database, so that we can show customers trends in behaviour, and identify hot-spots on specific repos - long build times, or spikes in failure rates.
+
+![Insights chart](https://pbs.twimg.com/media/FqnJ8rLXgAEnJDZ?format=png&name=large)
```
Comment (welteki) [inline posts/2023-03-31-managing-github-actions.md:103]: Maybe can use a newer picture of the insights with the updated tool tip that displays more information.

File: posts/2023-03-31-managing-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+```yaml
+- uses: self-actuated/hub-mirror@master
+```
+
+## Wrapping up
+
+I hope that you've enjoyed hearing a bit about our journey so far. With every new pilot customer we learn something new, and improve the offering.
+
+Whilst there was a significant amount of very technical at the beginning of actuated, most of our time now is spent on customer support, education, and improving the onboarding experience.
```
Comment (welteki) [inline posts/2023-03-31-managing-github-actions.md:225]: I think this has to be:

```suggestion
Whilst there was a significant amount of very technical work at the beginning of actuated, most of our time now is spent on customer support, education, and improving the onboarding experience.
```

File: posts/2023-03-31-managing-github-actions.md (hunk 1 above)
Comment (alexellis) [reply posts/2023-03-31-managing-github-actions.md:103]: I was thinking that too, but this one had the best data on the chart.

One thing we need to do separately is to add a section into the docs about the dashboard's insights tab.

File: posts/2023-03-31-managing-github-actions.md (hunk 2 above)
Comment (alexellis) [reply posts/2023-03-31-managing-github-actions.md:225]: yep :+1: thanks

**Pull request merged**